hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/shell/TestScmAdminHA.java

```diff
@@ -0,0 +1,66 @@
+package org.apache.hadoop.ozone.shell;
```

Comment:
Apart from UT, we can have an acceptance test to test CLI. 
You may find examples here: https://github.com/apache/hadoop-ozone/pull/375

Reply:
ack